lib/core/card-view/components/card-view-selectitem/card-view-selectitem.component.html

```diff
@@ -8,10 +8,14 @@
         <div *ngIf="isEditable()">
             <mat-form-field class="adf-select-item-padding-editable adf-property-value">
                 <mat-select [(value)]="value"
+                            panelClass="adf-select-filter"
                             (selectionChange)="onChange($event)"
                             data-automation-class="select-box">
+
+                    <adf-select-filter-input *ngIf="showInputFilter" (change)="onFilterInputChange($event)"></adf-select-filter-input>
+
                     <mat-option *ngIf="showNoneOption()">{{ 'CORE.CARDVIEW.NONE' | translate }}</mat-option>
-                    <mat-option *ngFor="let option of getOptions() | async"
+                    <mat-option *ngFor="let option of getOptions() | async | optionsFilter:filter"
```

Comment:
maybe `getOptions()` already provides the end result without the pipe?

Reply:
if we don't want a separation of operations i'm fine to change, your call